syntax = "proto3";

package mcp_bash;

// Service to execute bash commands
service BashExecutor {
  rpc ExecuteCommand (CommandRequest) returns (CommandReply) {}
}

// Request message containing the command to execute
message CommandRequest {
  string command = 1;
}

// Reply message containing the output of the command
message CommandReply {
  string stdout = 1;
  string stderr = 2;
  int32 return_code = 3;
}
